### PR TITLE
Add `symbols` field to standard model data

### PR DESCRIPTION
**Pull request by the-praxs**
All the providers return the `symbols` in the data as a comma-separated string.

### CODE REVIEW

**Comment by deeleeramone** [comment]
The other functions allowing multiple symbols accept a traditional Python List as well as a string-list.  Shouldn't this be the same here?

**Comment by the-praxs** [comment]
> The other functions allowing multiple symbols accept a traditional Python List as well as a string-list. Shouldn't this be the same here?

Its a separate issue and goes much deeper than this. The focus of this PR is to standardize the symbol fields, as was pointed out by you in the bug ticket.

I added that feature as a backlog ticket last week because this functionality should be universal across the codebase.

File: openbb_platform/providers/polygon/openbb_polygon/models/company_news.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class PolygonCompanyNewsQueryParams(CompanyNewsQueryParams):
     Source: https://polygon.io/docs/stocks/get_v2_reference_news
     """
 
-    __alias_dict__ = {"symbols": "ticker"}
+    __alias_dict__ = {"symbols": "tickers"}
```
Comment (deeleeramone) [inline openbb_platform/providers/polygon/openbb_polygon/models/company_news.py:22]: Polygon calls the parameter `ticker`, and it also doesn't support multiple symbols to the News endpoint.

Comment (the-praxs) [reply openbb_platform/providers/polygon/openbb_polygon/models/company_news.py:22]: Says `tickers` on the docs
image

**Comment by deeleeramone** [comment]
I get a validation error with `Ultima`:

```
obb.news.company(symbols="OXY", provider="ultima")
```

```
OpenBBError: 1 validation error for UltimaCompanyNewsData
symbols
  Field required [type=missing, input_value={'title': '... Occidental...slide', 'ticker': 'OXY'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.5/v/missing
    ```

**Comment by the-praxs** [comment]
> I get a validation error with `Ultima`:
> 
> ```
> obb.news.company(symbols="OXY", provider="ultima")
> ```
> 
>  ```
> OpenBBError: 1 validation error for UltimaCompanyNewsData
> symbols
>   Field required [type=missing, input_value={'title': '... Occidental...slide', 'ticker': 'OXY'}, input_type=dict]
>     For further information visit https://errors.pydantic.dev/2.5/v/missing
>     ```
> ```

I have not changed the Ultima provider file as we don't maintain it. The Ultima guys have been asked to separate their provider extension out from our repo into a different repo.

File: openbb_platform/providers/polygon/openbb_polygon/models/company_news.py (hunk 1 above)
Comment (the-praxs) [reply openbb_platform/providers/polygon/openbb_polygon/models/company_news.py]: Fixed.

Most providers support a string of comma-separated symbols so we are keeping it as `symbols`.

**Comment by deeleeramone** [comment]
> I have not changed the Ultima provider file as we don't maintain it. The Ultima guys have been asked to separate their provider extension out from our repo into a different repo.

It's because the 'symbols' field is now mandatory in the standard model, and they do not return it by this label.

**Comment by the-praxs** [comment]
> > I have not changed the Ultima provider file as we don't maintain it. The Ultima guys have been asked to separate their provider extension out from our repo into a different repo.
> 
> It's because the 'symbols' field is now mandatory in the standard model, and they do not return it by this label.

We are not responsible for their code and so they will have to fix it on their end. Ultima is neither maintained not shipped by us. 

You need to contact them to maintain their code.

**Comment by deeleeramone** [comment]
> You need to contact them and ask them to maintain their code on a separate repo.

It's a pretty simple fix to an error that is introduced by these changes.  What is our official policy for providing notification that breaking changes are happening?

**Comment by the-praxs** [comment]
> > You need to contact them and ask them to maintain their code on a separate repo.
> 
> It's a pretty simple fix to an error that is introduced by these changes. What is our official policy for providing notification that breaking changes are happening?

Please consult @jmaslek on this.

**Comment by deeleeramone** [comment]
> > > You need to contact them and ask them to maintain their code on a separate repo.
> > 
> > 
> > It's a pretty simple fix to an error that is introduced by these changes. What is our official policy for providing notification that breaking changes are happening?
> 
> Please consult @jmaslek on this.

They will be able to address it in conjunction with this PR #5795

**Pull request merged**